tests/typ/text/quotes.typ

```diff
@@ -13,6 +13,12 @@
 #set text(lang: "de", region: "CH")
 "Das Pferd frisst keinen Gurkensalat" war der erste jemals am 'Fernsprecher' gesagte Satz.
 
+#set text(lang: "es")
+"El caballo no come ensalada de pepino" fue la primera frase pronunciada por 'teléfono'.
+
+#set text(lang: "es", region: "MX")
+"El caballo no come ensalada de pepino" fue la primera frase pronunciada por 'teléfono'.
```

Comment:
You need to update the test image too, I believe there’s a VS Code tool somewhere in the repo.

Reply:
Done.